### PR TITLE
Added Logger, test includes minimum requirements

### PR DESCRIPTION
**Pull request by rhernand3z**
Includes the requirements as JSON string:

https://docs.google.com/document/d/1QidzMMNcXK0Su9wW_SpSb4lDIBB-39NqBl7brMBR0_g/edit

Test Output:

```
{"timestamp":"2017-06-22T20:41:17.866Z","level":"INFO","message":"testing logger","pid":"17812","jobId":"12345"}
```

### CODE REVIEW

File: src/helpers/Logger.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+const logger = new winston.Logger({
+  transports: [
+    new (winston.transports.Console)({
+      timestamp: () => {
+        return new Date().toISOString();
+      },
+      formatter: (options) => {
+        const result = {
+          timestamp: options.timestamp(),
+          level: options.level.toUpperCase()
```
Comment (kfriedman) [inline src/helpers/Logger.js]: Might be need at some point to add matching to our agreed-upon log levels (https://github.com/NYPL/engineering-general/issues/6). I noticed Winston uses slightly different levels (e.g. `WARN` vs `WARNING`): https://github.com/winstonjs/winston#logging-levels. I guess we haven't finalized the standard yet.

**Comment by rhernand3z** [comment]
Updated Logger to match agreed upon levels. See https://github.com/NYPL/engineering-general/issues/6

CC: @gkallenberg @holingpoon @kfriedman

File: src/helpers/Logger.js (hunk 1 above)
Comment (rhernand3z) [inline src/helpers/Logger.js]: Done.

**Comment by rhernand3z** [comment]
Fixes issue: #5

**Pull request merged**